cfgov/cfgov/settings/production.py

```diff
@@ -114,3 +115,10 @@
         'TIMEOUT': None,
     }
 }
+
+# ALLOWED_HOSTS should be defined as a JSON list in the ALLOWED_HOSTS
+# environment variable.
+try:
+	ALLOWED_HOSTS = json.loads(os.getenv('ALLOWED_HOSTS'))
```

Comment:
I notice these two lines are indented with tabs instead of spaces. It'd be nice to be consistent within a single file. Would you mind changing these to spaces?